transit_odp/browse/templates/browse/operators/operator_detail.html

```diff
@@ -357,6 +357,18 @@ <h3 class="govuk-heading-m govuk-!-margin-top-7">
           </div>
         </div>
       </div>
+    {% if is_fares_require_attention_active %}
+      <div class="govuk-grid-column-one-half">
+        <div class="feed-stat">
+          <span class="feed-stat__value">{{ fares_total_services_requiring_attention_percentage }}</span>
+          <span class="feed-stat__label">
+            {% filter force_escape %}
+              {% trans "Services requiring attention" %}
+            {% endfilter %}
+          </span>
+        </div>
+        {% endif %}
+      </div>
```

Comment:
All the stats show up when the flag is on instead of the percentage only (there is no end flag tag):

![image](https://github.com/user-attachments/assets/4e26ec6a-53ae-4e65-8fe1-625f6cec714d)